### PR TITLE
Black 21.4b0

### PR DESCRIPTION
**Pull request by dgasmith**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dgasmith** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {% set name = "black" %}
-{% set version = "20.8b1" %}
+{% set version = "21.4b1" %}
```
Comment (xhochy) [inline recipe/meta.yaml:2]:
```suggestion
{% set version = "21.4b0" %}
```

Comment (dgasmith) [reply recipe/meta.yaml]: Thanks! - You caught me in mid edit there as well.

**Comment by dgasmith** [comment]
@xhochy Should be good to go.

**Comment by xhochy** [comment]
Thanks!

**Pull request merged**